kasa/smart/modules/batterysensor.py

```diff
@@ -43,6 +43,10 @@ def _initialize_features(self):
                 )
             )
 
+    def query(self) -> dict:
+        """Query to execute during the update cycle."""
+        return {}
+
```

Comment:
A lot of the hub child devices appear to incorrectly expose a module query even though they don't call/access any modules. In order to support the `self.data` check in this PR they need to expose an empty query dict.